CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ Latest
 
 ### Language
 
+* (!) The following keywords are now reserved identifiers:
```

Comment:
What is the implication for users of these being reserved? If it's "_ is added in transpiled names" then that seems problematic, as we see in the test examples: you give your class or package a name is and it's mutated in the public API. Better would be to just disallow these in code, perhaps?